### PR TITLE
reference/sql: make recover table notice obvious and clear

### PR DESCRIPTION
**Pull request by GregoryIan**
### What is the related PR or file link(s)? 

Make the notice of recover table more conspicuous and clear

### Which version does your change affect? 
3.0 

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/reference/sql/statements/recover-table.md [hunk 1]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
 RECOVER TABLE BY JOB ddl_job_id
 ```
 
+## **注意事项**
```
Comment (lilin90) [inline dev/reference/sql/statements/recover-table.md]: Do you really want to apply the 2nd-level heading here? If not, please use the note format.

File: v3.0/reference/sql/statements/recover-table.md [hunk 2]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
 RECOVER TABLE BY JOB ddl_job_id
 ```
 
+## **注意事项**
```
Comment (lilin90) [inline v3.0/reference/sql/statements/recover-table.md]: ditto

File: dev/reference/sql/statements/recover-table.md (hunk 1 above)
Comment (GregoryIan) [reply dev/reference/sql/statements/recover-table.md]: yep

Comment (lilin90) [reply dev/reference/sql/statements/recover-table.md]: Which one? 2nd-level heading or a Note?

Comment (lilin90) [inline dev/reference/sql/statements/recover-table.md]:
```suggestion
## 注意事项
```

File: dev/reference/sql/statements/recover-table.md [hunk 3]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
...
+>
```
Comment (lilin90) [inline dev/reference/sql/statements/recover-table.md]: Please remove all `>` characters.

File: dev/reference/sql/statements/recover-table.md [hunk 4]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
 RECOVER TABLE BY JOB ddl_job_id
 ```
 
+## 注意事项
+
+如果删除表后并过了 GC life time，就不能再用 RECOVER TABLE 来恢复被删除的表了，执行 RECOVER TABLE 语句会返回类似错误：`snapshot is older than GC safe point 2019-07-10 13:45:57 +0800 CST`。
```
Comment (lilin90) [inline dev/reference/sql/statements/recover-table.md]:
```suggestion
如果删除表后并过了 GC life time，就不能再用 `RECOVER TABLE` 来恢复被删除的表了，执行 `RECOVER TABLE` 语句会返回类似错误：`snapshot is older than GC safe point 2019-07-10 13:45:57 +0800 CST`。
```

File: dev/reference/sql/statements/recover-table.md [hunk 5]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
...
+
+对于 3.0.0 及之后的 TiDB 版本，不推荐在使用 binlog 的情况下使用 recover table 功能。
```
Comment (lilin90) [inline dev/reference/sql/statements/recover-table.md]:
```suggestion
对于 3.0.0 及之后的 TiDB 版本，不推荐在使用 binlog 的情况下使用 `RECOVER TABLE` 功能。
```

File: dev/reference/sql/statements/recover-table.md [hunk 6]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
...
+
+binlog 在 3.0.1 支持 recover table 后，可在下面的情况下使用 recover table：
```
Comment (lilin90) [inline dev/reference/sql/statements/recover-table.md]:
```suggestion
Binlog 在 3.0.1 支持 `RECOVER TABLE` 后，可在下面的情况下使用 `RECOVER TABLE`：
```

File: dev/reference/sql/statements/recover-table.md [hunk 7]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
...
+
+* 3.0.1+ 版本 binlog
+* 主从集群都使用 TiDB 3.0
+* 从集群 GC lifet time 一定要长于主集群（不过由于上下游同步的延迟，可能也会造成下游 recover 失败）
+
+### TiDB Binlog 同步错误处理
+
+当使用 TiDB Binlog 同步工具时，上游 TiDB 使用 RECOVER TABLE 后，TiDB Binlog 可能会因为下面几个原因造成同步终端：
```
Comment (lilin90) [inline dev/reference/sql/statements/recover-table.md]:
```suggestion
当使用 TiDB Binlog 同步工具时，上游 TiDB 使用 `RECOVER TABLE` 后，TiDB Binlog 可能会因为下面几个原因造成同步终端：
```

File: dev/reference/sql/statements/recover-table.md [hunk 8]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
...
+
+* 下游数据库不支持 RECOVER TABLE 语句。 类似错误： `check the manual that corresponds to your MySQL server version for the right syntax to use near 'RECOVER TABLE table_name'`
```
Comment (lilin90) [inline dev/reference/sql/statements/recover-table.md]:
```suggestion
* 下游数据库不支持 `RECOVER TABLE` 语句。类似错误：`check the manual that corresponds to your MySQL server version for the right syntax to use near 'RECOVER TABLE table_name'`。
```

File: dev/reference/sql/statements/recover-table.md [hunk 9]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
...
+
+* 上下游数据库的 GC life time 不一样。 类似错误： `snapshot is older than GC safe point 2019-07-10 13:45:57 +0800 CST`。
```
Comment (lilin90) [inline dev/reference/sql/statements/recover-table.md]:
```suggestion
* 上下游数据库的 GC life time 不一样。类似错误：`snapshot is older than GC safe point 2019-07-10 13:45:57 +0800 CST`。
```

File: dev/reference/sql/statements/recover-table.md [hunk 10]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
...
+
+* 上下游数据库的同步延迟。类似错误：`snapshot is older than GC safe point 2019-07-10 13:45:57 +0800 CST`。
+
+只能通过重新 [全量导入被删除的表](/v3.0/how-to/migrate/overview.md#mysql-数据的全量迁移) 来恢复 TiDB Binlog 的数据同步
```
Comment (lilin90) [inline dev/reference/sql/statements/recover-table.md]:
```suggestion
只能通过重新[全量导入被删除的表](/how-to/migrate/overview.md#mysql-数据的全量迁移)来恢复 TiDB Binlog 的数据同步。
```

File: dev/reference/sql/statements/recover-table.md [hunk 11]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
...
+
+Binlog 在 3.0.1 支持 `RECOVER TABLE` 后，可在下面的情况下使用 `RECOVER TABLE`：
+
+* 3.0.1+ 版本 binlog
+* 主从集群都使用 TiDB 3.0
+* 从集群 GC lifet time 一定要长于主集群（不过由于上下游同步的延迟，可能也会造成下游 recover 失败）
+
+### TiDB Binlog 同步错误处理
+
+当使用 TiDB Binlog 同步工具时，上游 TiDB 使用 `RECOVER TABLE` 后，TiDB Binlog 可能会因为下面几个原因造成同步终端：
```
Comment (july2993) [inline dev/reference/sql/statements/recover-table.md]: s/终端/中断

File: v3.0/reference/sql/statements/recover-table.md [hunk 12]
```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
...
+
+Binlog 在 3.0.1 支持 `RECOVER TABLE` 后，可在下面的情况下使用 `RECOVER TABLE`：
+
+* 3.0.1+ 版本 binlog
+* 主从集群都使用 TiDB 3.0
+* 从集群 GC lifet time 一定要长于主集群（不过由于上下游同步的延迟，可能也会造成下游 recover 失败）
+
+### TiDB Binlog 同步错误处理
+
+当使用 TiDB Binlog 同步工具时，上游 TiDB 使用 `RECOVER TABLE` 后，TiDB Binlog 可能会因为下面几个原因造成同步终端：
```
Comment (july2993) [inline v3.0/reference/sql/statements/recover-table.md]: s/终端/中断

**Pull request merged**